src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -828,16 +828,16 @@ private void RunGarbageCollection(object state)
             if (deactivatedActors.Count > 0)
             {
 #pragma warning disable 4014
-                // let the thread continue and deactivate the actors 
-                // in asynchronous manner and then reset the GC timer
+                // Lets the thread continue and deactivate the actors 
+                // in asynchronous manner and then reset the GC timer.
                 this.DeactivateActorsAsync(deactivatedActors);
 #pragma warning restore 4014
             }
 
             this.ArmGcTimer();
         }
 
-        // ReSharper disable once UnusedMethodReturnValue.Local
+        // Disables the ReSharper once UnusedMethodReturnValue.Local.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.